modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/test/java/com/liferay/dynamic/data/mapping/model/DDMFormInstanceSettingsTest.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.model;
+
+import com.liferay.dynamic.data.mapping.test.util.BaseDDMFormFieldTypeSettingsTestCase;
+import com.liferay.dynamic.data.mapping.util.DDMFormFactory;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Pedro Leite
+ */
+public class DDMFormInstanceSettingsTest
+	extends BaseDDMFormFieldTypeSettingsTestCase {
```

Comment:
Can you rename BaseDDMFormFieldTypeSettingsTestCase to Base**SettingsTest**Case ?

Since we want to use it with DDMFormInstance**SettingsTest** and *DDMFormFieldType**SettingsTest**.